shell/platform/common/client_wrapper/event_channel_unittests.cc

```diff
@@ -33,7 +33,7 @@ class TestBinaryMessenger : public BinaryMessenger {
     return last_message_handler_channel_;
   }
 
-  BinaryMessageHandler last_message_handler() { return last_message_handler_; }
+  BinaryMessageHandler& last_message_handler() { return last_message_handler_; }
```

Comment:
I don't believe this has to be a non-const reference.

Reply:
Interesting, this seems to work. My intuition was that a const reference would force a copy of the lambda, but the tests still pass. Will add!

Reply:
Updated